ui/lib/types/mcp.ts

```
@@ -1,3 +1,5 @@
+import { Function } from "./logs";
```

Biome (2.1.2)

[error] Do not shadow the global "Function" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)